tests/rule_based_profiler/bobby_user_workflow_fixture.py

```diff
@@ -144,14 +151,14 @@ def bobby_columnar_table_multi_batch():
                         }
                     }
                 },
-            },
+            }
         ),
         ExpectationConfiguration(
             **{
                 "kwargs": {
                     "column": "trip_distance",
-                    "min_value": 21.42,
-                    "max_value": 74.05,
+                    "min_value": 10.52,
+                    "max_value": 84.95,
```

Comment:
I haven't looked at the data, but it's surprising to me that our bounds are so much looser in this fixture than the previous one...unless we only look at two data points so we're saying we have super low confidence. Is that the case?

Reply:
@jcampbell As a general guide to the unit tests, please note that only the "Bobster" use case relies on the probabilistic, "bootstrap" sampling method; all the other tests use the "oneshot" (deterministic) sampling method, which we know to be inaccurate.  While the "bootstrap" training set (Bobster use case) uses 35 months of data, with the number of rows normally distributed with the mean of 5,000 and the standard deviation of 1,000 rows, the "oneshot" training set consists of just 2 months with 7,500 and 8,500 rows (i.e., the mean of 8,000 rows).

One of the reasons for the large spread is that the tests use the false positive rate of `0.01`.  This translates into `2.6` standard deviations around the mean needed to achieve the coverage.  Happy to discuss what we can improve here.

Thanks.

Reply:
Sounds like no change needed -- thanks for the thorough explanation!